### ISSUE TITLE
Creating a new TornadoFX project defaults to a com.example.demo package/classpath.

### ISSUE THREAD

**Issue by pdichone** [edvin/tornadofx#1175]
Is there a way to add our own package name (domain) in the project creating process (using intellij wizard)?  Right now, the only way to accomplish this is by refactoring.

Is this feature coming soon?

Thanks

**Comment by edvin** [comment]
No, that's not supported by the plugin at the moment and nobody is working on that feature I'm afraid. You might give it a go yourself if you have time and want to help out :)
